### DISCUSSION TITLE
你相信外星人嗎👽？

### DISCUSSION THREAD

**Discussion by pudding0803** [1101-GS4719-C/Python-Community-GS4719-C#1469] [General]
## 新聞

今天看到一篇報導：https://www.chinatimes.com/realtimenews/20230530001696-260408?chdtv
（其實我是在查 AI 的時候不小心查到的🤣）

內容大致為：
> 為美國政府研究幽浮（UFO）的史丹佛大學（Stanford University）醫學院病理學教授諾蘭（Garry Nolan）本周在紐約市舉行的「五角大廈、外星人和墜毀的幽浮」（The Pentagon, Extraterrestrial Intelligence, and Crashed UFOs.）全球領導階層和網絡論壇中，發表了驚人的言論說，他認為外星人不僅百分之百造訪過地球，甚至還和人類共存了幾千年。

## 我的觀點

先說結論，我目前是不信的👽

但並不是指不相信在地球之外有其他物種的存在，而是像是報導中──抑或是其他文章中──一直提及的「觀察」與「共存」

如果真的有外星人，並像是普遍大眾認為的：具有非常高的科技，那他們一直花時間跑來有什麼意義？而且為何還要一直躲躲藏藏？更甚至為何一定長得類似人類（就跟為何大部分對鬼👻的想像都是人的形體）？這樣是不是顯得有點自以為是🤔

## 討論

然後我問了 ChatGPT 🥴
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/45879162/7e9272f5-c8cf-4bb1-8420-96bebfc67e34)

因為目前對於外星人也沒什麼證據之類的，所以歡迎大家隨便猜測、隨便假設、隨便討論🥹

**Comment by Tzuhsuannnnn** [comment]
我相信有外星人，因為我認為世界上那麼多星系，應該不只有地球有生物居住，在世界的其他角落應該也有生物或其他城市文明存在，只是我們人類還沒有能力觀察到，也許其實他們的科技文明已經有辦法觀測到我們並每天觀察我們也說不定。

**Comment by roma0323** [comment]
信

**Comment by Aaron0810** [comment]
我也是偏向相信有外星人的，如同1樓所說，宇宙中星系那麼多，總是會有另外一個星球是適合生物居住的，但我認為那些外星生物不會像電影演的一樣那麼擬人，反正有可能在視覺上是會顛覆我們人類的審美觀的，甚至在維度或向量方面也會完全不一樣，但總之我也是願意相信有外星人存在的。

**Comment by Alexgns** [comment]
地球僅僅是宇宙中極為微小的一個存在，我相信在宇宙中的其他地方，必定存在各式各樣不同的物種，畢竟不太可能地球為那億萬中選一唯一有各式物種存在的地方

**Comment by alstinlin** [comment]
我也相信有外星人的存在，但不一定擁有很先進的科技或很高的智慧，有可能就像人類一樣，至於長相我覺得會和電影或小說描寫的差很多，因為在跟地球完全不同的環境下，長相特徵跟人類一定會十分不同

**Comment by sumairu0727** [comment]
根據宇宙各向同性的看法，地球絕對不會是唯一如此特別的星球，所以我相信有外星人的存在，只是可跟地球的生命大有不同。但我認為人類不應該向外探索外星人，如過遇到比我們科技能力更高的生命體可能會被他們殖民。

**Comment by Miinteaa** [comment]
宇宙的範圍如此之大，也許在別的星系也有符合物種生存條件、類似地球環境的星球。所以我也偏向支持有外星物種的論點！
但其是否登陸過地球還是有待討論，因為到現在為止都沒有確切的證據可以證明外星物種存在，也就是費米悖論。
如果今天外星物種智慧高於人類，那麼為何沒有任何他們探索星系，抑或者是征服、殖民的痕跡呢？

**Comment by Superlan8** [comment]
我覺得外星人是存在的，畢竟宇宙那麼大，有那麼多的星系，有很大的機率存在可以讓生物生存的星球，但我認為他們目前的智慧可能最高跟人類差不多而已，不會像科幻電影中那樣，超級猛，要不然的話，他們應該會發現我們的存在。

**Comment by Brownnnn78** [comment]
我相信外星人是存在的，感覺很好玩，之前很喜歡看老高的影片，其中還有講到地球是用來關罪犯靈魂的地方，靈魂死掉之後也離不開地球，而且地球對於宇宙真的很小，人絕對不是最高等的物種。我猜。

**Comment by WillChen110403039** [comment]
我是相信有外星人地存在的，畢竟宇宙如此廣大，若只有人類存在也太不合理，只是我覺得外星人的樣貌可能並非為人類一直以來的認知，可能超出人類的想像，因為五官或皮膚之類的特徵都只是動物的特徵並無法套用在整個宇宙中。

**Comment by ashleyy-huang** [comment]
現今還沒有確切的證據指向有外星人的存在，但這也並不代表沒有外星人的存在，我依舊保持著好奇的心態來看待這件事，因為現在科技越來越發達，什麼事情都有可能會發生，來打破我們現有的觀念與思想。

**Comment by lyien1021** [comment]
好意外看到大家都相信有外星人的存在，不過我自己其實也是偏向相信的，但應該不會跟很多外星電影裡的外星人一樣噁心啦（希望），其實蠻期待有一天科技人類能跟外星生物交流。

**Comment by JessyCheng2003** [comment]
我自己也相信有外星人 一部分可能是因為之前看了「老高」的影片 再加上我覺得宇宙那麼大 有那麼多星球 不會只有地球有生物 我認為之所以外星人很少到訪地球 可能是因為我們的環境不適合他們生存

**Comment by brian431** [comment]
我認為是有的，我不認為在如此浩大的宇宙中只有地球剛好有演化出智慧生物的條件，只是距離真的太遠了而且會一直變得更遠。所以到幾億年後也不一定就能碰到他們。

**Comment by xiang305** [comment]
我也是相信有外星人的，就跟前面幾樓的想法一樣，宇宙這麼大，應該還是存在另一顆適合居住的星球。雖然目前仍然沒有確切的證據證明，不過我認為這或許是人類與外星人雙方都不敢貿然與對方接觸吧，要是一個不小心自己的星球就被對方破壞了。我記得之前也有專家呼籲大家不要嘗試去尋找外星人吧

**Comment by MaxChi870** [comment]
我認為宇宙中應該還有其他具有智慧意識的生物，只是目前尚未有所發現罷了，不過維持現狀或許會是最好的狀態，畢儘雙方的接觸對於彼此而言一定會有極大的影響與變化。

**Comment by BOSENCHANG2004** [comment]
目前還沒有確鑿證據能證明外星智慧生命的存在，雖然有很多UFO目擊報告、軍方影片、甚至NASA或SETI持續在找尋外星文明的訊號（像是窄頻無線電訊號），但都沒法證明「他們來過」或「他們正在看我們」。
我個人推論是：
有外星生命的機率超高，
但要是像我們這樣有科技、有文明的「智慧外星人」，而且還能跟我們溝通、互動，那可能就稀有多了。
